helm/charts/hpe-csi-driver/README.md

```diff
@@ -32,7 +32,7 @@ The following table lists the configurable parameters of the chart and their def
 | logLevel                  | Log level. Can be one of `info`, `debug`, `trace`, `warn` and `error`. | info             |
 | registry                  | Registry to pull HPE CSI Driver container images from.                 | quay.io          |
 | kubeletRootDir            | The kubelet root directory path.                                       | /var/lib/kubelet |
-| cspClientTimeout          | CSP client timeout for HPE Alletra 9000, Primera and 3PAR (60-360 sec).| 60               |
+| csiAttacherTimeout        | CSI Attacher timeout (in seconds).                                     | 15s - 180s    |
```

Comment:
we don't need these parameter in helm chart as per the discussion @pavansshanbhag can you help @maddiegos with the explanation

Reply:
done